packages/solo/src/vat-http.js

```diff
@@ -104,18 +104,23 @@ export function buildRootObject(vatPowers) {
 
   const { promise: walletP, resolve: resolveWallet } = makePromiseKit();
   const { promise: attMakerP, resolve: resolveAttMaker } = makePromiseKit();
-  Promise.all([walletP, attMakerP]).then(async ([wallet, attMaker]) => {
-    const walletAdmin = await E(wallet).getAdminFacet();
-    // console.debug('introduce', { wallet, walletAdmin, attMaker });
-    E(walletAdmin).resolveAttMaker(attMaker);
-  });
+
+  const resolvedAttrMakers = Promise.all([walletP, attMakerP]).then(
```

Comment:
It's not an *Attr*ibuteMaker, it's an *Att*estationMaker.  And together, they are `const walletInitializedP`.